app/src/androidTest/java/org/dhis2/usescases/flow/teiFlow/TeiFlowRobot.kt

```diff
@@ -71,8 +71,9 @@ class TeiFlowRobot : BaseRobot() {
         }
 
         teiDashboardRobot {
-            checkLockCompleteIconIsDisplay()
-            checkCanNotAddEvent()
+            // Needs to be refactor to check new Infobar
+            //checkLockCompleteIconIsDisplay()
+            //checkCanNotAddEvent()
```

Comment:
Please, can you create an issue of type task and set in ready to start for 2.9 to implement all the tests that has been commented?

Reply:
done